<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="mainStackedWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>88</y>
      <width>782</width>
      <height>361</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="audioPage">
     <widget class="QWidget" name="layoutWidget00">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>781</width>
        <height>261</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="audGLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="audRecordButn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Record</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="audPlayButn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Play</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="audPauseButn">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Pause</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="audLevelLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Audio Level:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <layout class="QHBoxLayout" name="audLevelHLayout"/>
       </item>
      </layout>
     </widget>
     <widget class="QCheckBox" name="audSourceCBox">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>10</y>
        <width>150</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string>Received Audio</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="contactPage">
     <widget class="QStackedWidget" name="cntStackedWidget">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>621</width>
        <height>341</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="cntReceivePage">
       <widget class="QWidget" name="layoutWidget01">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>10</y>
          <width>541</width>
          <height>221</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="cntContactGLayout">
         <item row="1" column="0" colspan="2">
          <layout class="QGridLayout" name="cntNameGLayout">
           <item row="0" column="0" rowspan="2">
            <widget class="QLabel" name="cntNameLabel">
             <property name="text">
              <string>Name:      </string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="cntNameLEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="cntNameCbBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLabel" name="cntRecResultLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <layout class="QGridLayout" name="cntGroupGLayout">
           <item row="0" column="0" rowspan="2">
            <widget class="QLabel" name="cntGroupLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Group:     </string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="cntGroupLEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="cntGroupCbBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget02">
        <property name="geometry">
         <rect>
          <x>141</x>
          <y>239</y>
          <width>441</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="cntRecProgressHLayout"/>
       </widget>
       <widget class="QLabel" name="cntRecProgressLabel">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>240</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Progress:</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget03">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>290</y>
          <width>541</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="cntButnHLayout">
         <item>
          <widget class="QPushButton" name="cntDeleteButn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cntCancelButn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cntAcceptButn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Accept</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="cntSendPage">
       <widget class="QWidget" name="layoutWidget06_2">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>140</y>
          <width>441</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="cntSendProgressHLayout"/>
       </widget>
       <widget class="QLabel" name="cntSendProgressLabel">
        <property name="geometry">
         <rect>
          <x>29</x>
          <y>141</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Progress:</string>
        </property>
       </widget>
       <widget class="QLabel" name="cntSendResultLabel">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>65</y>
          <width>539</width>
          <height>69</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="cntReceiveButn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>120</width>
        <height>120</height>
       </rect>
      </property>
      <property name="text">
       <string>Receive</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cntSendButn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>120</width>
        <height>120</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="settingPage">
     <widget class="QStackedWidget" name="setStackedWidget">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>621</width>
        <height>341</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="setSerialPage">
       <widget class="QGroupBox" name="setParametersBox">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>20</y>
          <width>361</width>
          <height>261</height>
         </rect>
        </property>
        <property name="title">
         <string>Select Parameters</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="5" column="1">
          <widget class="QComboBox" name="setFlowControlCbBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="setParityCbBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="setStopBitsCbBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="setDataBitsCbBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="setStopBitsLabel">
           <property name="text">
            <string>Stop bits:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="setFlowControlLabel">
           <property name="text">
            <string>Flow control:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="setParityLabel">
           <property name="text">
            <string>Parity:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="setBaudRateLabel">
           <property name="text">
            <string>Baud Rate:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="setDataBitsLabel">
           <property name="text">
            <string>Data bits:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="setBaudRateCbBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="setSerialPortInfoLBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="setSerialPortLabel">
           <property name="text">
            <string>Serial Port:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QPushButton" name="setApplyButn">
        <property name="geometry">
         <rect>
          <x>405</x>
          <y>291</y>
          <width>176</width>
          <height>39</height>
         </rect>
        </property>
        <property name="text">
         <string>Apply</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="setAboutPage">
       <widget class="QLabel" name="pictureLabel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>621</width>
          <height>341</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="setSerialButn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>120</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string>Serial</string>
      </property>
     </widget>
     <widget class="QPushButton" name="setOffButn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>240</y>
        <width>120</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string>Turn off</string>
      </property>
     </widget>
     <widget class="QPushButton" name="setAboutButn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>120</width>
        <height>80</height>
       </rect>
      </property>
      <property name="text">
       <string>About</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="messagePage">
     <widget class="QStackedWidget" name="msgStackedWidget">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>621</width>
        <height>341</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="msgSendPage">
       <widget class="QWidget" name="layoutWidget04">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>10</y>
          <width>541</width>
          <height>101</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="msgContactGLayout">
         <item row="1" column="0" colspan="2">
          <layout class="QGridLayout" name="msgNameGLayout">
           <item row="0" column="0" rowspan="2">
            <widget class="QLabel" name="msgNameLabel">
             <property name="text">
              <string>Name:      </string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="msgNameCbBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="0" colspan="2">
          <layout class="QGridLayout" name="msgGroupGLayout">
           <item row="0" column="0" rowspan="2">
            <widget class="QLabel" name="msgGroupLabel">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Group:     </string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="msgGroupCbBox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="msgSendResultLabel">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>160</y>
          <width>539</width>
          <height>69</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget02_2">
        <property name="geometry">
         <rect>
          <x>141</x>
          <y>239</y>
          <width>441</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="msgSendProgressHLayout"/>
       </widget>
       <widget class="QLabel" name="msgSendProgressLabel">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>240</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Progress:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="msgTransmitButn">
        <property name="geometry">
         <rect>
          <x>405</x>
          <y>291</y>
          <width>176</width>
          <height>39</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Transmit</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="msgReceivePage">
       <widget class="QLabel" name="msgRecProgressLabel">
        <property name="geometry">
         <rect>
          <x>29</x>
          <y>141</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Progress:</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget06">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>140</y>
          <width>441</width>
          <height>41</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="msgRecProgressHLayout"/>
       </widget>
       <widget class="QLabel" name="msgRecResultLabel">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>65</y>
          <width>539</width>
          <height>69</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="msgSendButn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>120</width>
        <height>120</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QPushButton" name="msgReceiveButn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>120</width>
        <height>120</height>
       </rect>
      </property>
      <property name="text">
       <string>Receive</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget07">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>781</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="selectionHLayout">
     <item>
      <widget class="QPushButton" name="selAudioButn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Audio</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selTextButn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Text</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selContactsButn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Contacts</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selMessagesButn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Messages</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selSettingsButn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Settings</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="messageLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>730</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
